stable/kube-state-metrics/templates/clusterrole.yaml

```diff
@@ -170,6 +170,10 @@ rules:
   resources:
     - storageclasses
   verbs: ["list", "watch"]
```

Comment:
We probably need to remove this rule(L169-L172). `"storageclasses.k8s.io"` is thw wrong apiGroup anyway

Reply:
Done